### ISSUE TITLE
Chat room

### ISSUE THREAD

**Issue by MC-Railman** [jbg35/Project3_Bioinformatics#3]
Sup guys. I was out of town yesterday thanks to some health stuff that came up- apologies I couldn't help out yesterday. 

I saw on Jake's branch he mentioned some kind of chat room, and I think this might work well enough? Idk if you'll all get notifications from here unless you assign yourself to them, but eh we can see. I'm here to help out wherever I can all day today.

**Comment by AnthonyScaglione** [comment]
Lol probably close enough. Anyways I still don't have the two fold thing for the preprocessing in, can anyone explain what exactly needs done there?

**Comment by MC-Railman** [comment]
From what I'm able to figure out from her wording.. Each column is a separate experiment. I'm going to use the little table under her Part-1 section as an example.

If you look at the second entry, AF007551_at, for ALL experiments it has the values of 141, 87, and 244. Under the two fold change part she has "(max/min <2)" written. I'm interpreting that as take the max and divide by min, and if the result is less than 2 eliminate it.

For AF007551_at ALL values, it would be 244/87, which is around 2.8 so it should be safe. If you look at AF008937_at is has 339, 403, and 440 for ALL. 440/339 = 1.29, so it would be removed. That's how I'm looking at that at least..

**Comment by jbg35** [comment]
Quick question about the feature selection. So do we only pick 2 values (1 ALL column and 1 AML column) to compare using the classifier? Rather than using all (27 ALL and 11 AML) values.

**Comment by MC-Railman** [comment]
I think all values. Based on her wording of "Syntax: TTEST(array1,array2,tails,type)", she has the first two variables as an array. I'm guessing that's array1 for ALL and arrray2 for AML

**Comment by MC-Railman** [comment]
Also- I can't get KNNclassifier.py to build on my computer. It gets stuck on the ".drop(columns=" lines. It errors out saying the last column "is not found in axis" (in the file). Is this the same for you both, or is my computer just being special?

**Comment by jbg35** [comment]
Works on mine. Are you on windows or linux? might have something to do with how the file is read.

**Comment by jbg35** [comment]
Oh, I think i just fixed that problem. try pulling the latest version

**Comment by MC-Railman** [comment]
I'm on Windows 10. Just pulled the "KNN classifier kinda done" commit and I'm getting the same problem. It might be a Windows thing :(

**Comment by jbg35** [comment]
![2019-05-05_14-01](https://user-images.githubusercontent.com/45011876/57198221-55398800-6f3e-11e9-8a4a-9ca3db4dda9a.png)
This is how it comes out on mine. The top one is the vector that is being used to train and the bottom is after putting the leuk file and having the kNN algorithm predict what each feature is.

**Comment by jbg35** [comment]
I might have fixed it. Try pulling now

**Comment by MC-Railman** [comment]
![image](https://user-images.githubusercontent.com/47323720/57198311-2839a500-6f3f-11e9-9704-85706d32dfc5.png)

No luck.. I can try and play around with it some more

**Comment by jbg35** [comment]
I think it's the way windows represents new lines. For linux \n works. Try doing a print(top50) before that line. On that table there might be a character following P-VALUE on the first row. Use that value after 'P-VALUE(character)'.

**Comment by jbg35** [comment]
All that line is trying to do is remove the Acession and P-VALUE column. So if you can't get it working you could just delete those columns in the actual testoutputTop50Ascending file.

**Comment by MC-Railman** [comment]
I hate Windows 10.. it had /r/n after P-VALUE. Its building now.. lol

**Comment by jbg35** [comment]
Yeah, I think that's just a windows thing where they represent the \ characters differently.

**Comment by MC-Railman** [comment]
I've never had to use \r before \n characters before in any other project :') Gosh.. Well okay I'm late to the party, thanks Windows.. What all can I help you with right now?

**Comment by jbg35** [comment]
https://docs.google.com/presentation/d/1JA-Bg3dwiFgVDu_gcSBwT7bPF2Cbk40JTuFpXy9-TKc/edit?usp=sharing
heres a google slides for the presentation.

**Comment by AnthonyScaglione** [comment]
There is an Ubuntu shell in the windows 10 store. Just download it and it runs an isolated linux environment within windows, its what I use to test all my stuff. Should solve your problem https://tutorials.ubuntu.com/tutorial/tutorial-ubuntu-on-windows#0

**Comment by AnthonyScaglione** [comment]
I dont have edit access to the powerpoint

**Comment by MC-Railman** [comment]
We should have access now @AnthonyScaglione 

I just added different cases for the KNN for 6a-6e on the project website and committed the changes. I slapped in an extra 6f for the best/most accurate combo I found when I was playing around with the params of the knn.
